helm/node-exporter-app/values.yaml

```diff
@@ -23,6 +23,9 @@ env: {}
 ##  env:
 ##    VARIABLE: value
 
+serviceMonitor:
+  namespace: monitoring
```

Comment:
I'm not 100% sure that `monitoring` namespace exist by default on WC

Reply:
That's my issue here

Reply:
Does the PR create the namespace if it doesn't exist ?

Reply:
Unfortunately no. But let's change the namespace to `giantswarm` as proposed by @hervenicol